gr-filter/lib/gen_interpolator_taps/objective_fct.c

```diff
@@ -49,7 +51,7 @@ double		global_B  = B;
  */
 
 static double
-integrand (double omega)
+integrand (double omega, void *params)
```

Comment:
Won't compilers complain about the unused param? Can you just do `integrand(double omega, void*)`?

Reply:
Yeah probably, but I didn't set the compiler switches in generate.sh to gripe about uninitialized variables, since most of the code was existing.

A better thing to do would be to use the "params" pointer to pass in a structure of data to eliminate the global_n and global_h variables.  Probably more work than its worth though.